src/transformers/models/mask2former/modeling_mask2former.py

```diff
@@ -2149,70 +2168,82 @@ def _init_weights(self, module: nn.Module):
         xavier_std = self.config.init_xavier_std
         std = self.config.init_std
```

Comment:
This too vars are not used anymore, you just need to remove them to make it style CI fixed